uwblueprint/headsupguys
# [Module Experience] Breakpoints

## Notion ticket link

[[Module Experience] Breakpoints](https://www.notion.so/uwblueprintexecs/Module-Experience-Breakpoints-f5c6b0b1e8114b218f4e9724796b026c)



## Implementation description
* Fixed the responsive design for the /module/[moduleID] page
* Allowed interaction with the module/[moduleID] page



## Steps to test
1. Make a module and take note it it's generated ID in the url (after saving)
2. Navigate to localhost::3000/module/moduleID
3. Ensure that the module looks like it did in the preview
4. Ensure that you can check any multiplechoice/multiselect boxes, and write in any appropriate text fields in the moduleID page but not on the module creation page
5. Check for any new bugs on the module creation and module viewing pages and tell @ChinemeremChigbo if any are found 



## What should reviewers focus on?
* The module interaction page


## Checklist
